zovis-client/src/components/App.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { Container } from "react-bootstrap";
+import { AuthProvider } from "../contexts/AuthContext";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import Dashboard from "./Dashboard";
+import Users from "./Users";
+import NewQr from "./NewQr";
+import Qrs from "./Qrs";
+import Login from "./Login";
+import PolicyOrgNew from "./PolicyOrgNew";
+import PolicySearch from "./PolicySearch";
+import PoliciesIndex from "./PoliciesIndex";
+import Entitlement from "./Entitlement";
+import NotFound from "./NotFound";
+import PrivateRoute from "./PrivateRoute";
+import { Role } from "../types/user";
+import ForgotPassword from "./ForgotPassword";
+import { logger } from "../utils/logger";
+
+function App() {
+  logger.info("v 6.27.22");
+
+  return (
+    <Container
+      className="d-flex align-items-start justify-content-center pt-4"
+      style={{ minHeight: "100vh" }}
+    >
+      <div className="w-100">
+        <Router>
+          <AuthProvider>
+            <Switch>
+              <PrivateRoute
```

Comment:
Misschien overwegen om de routes lazy te maken zodat ze dynamisch ingeladen kunnen worden o.b.v van de route.

Reply:
Schilt in de main bundle size

Reply:
[https://reactjs.org/docs/code-splitting.html](https://reactjs.org/docs/code-splitting.html)